contracts/fast/lib/AFastFacet.sol

```diff
@@ -69,6 +71,17 @@ abstract contract AFastFacet is IFastEvents {
     _;
   }
 
+  /** @dev Ensures a candidate is active.
+   *  @param candidate The address to check activation status on.
+   */
+  modifier onlyExchangeActiveMember(address candidate) {
+    require(
+      IHasActivations(LibFast.data().exchange).isMemberActive(candidate),
```

Comment:
Very nice :) That's exactly it! Good job using the `IHasActivations` interface here, and remaining agnostic of which facet implements it. :+1: :+1: :+1: